php/data/mapper.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Thie file defines the \LORIS\Data\Mapper interface.
+ *
+ * PHP Version 7
+ *
+ * @category   Data
+ * @package    Main
+ * @subpackage Data
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+namespace LORIS\Data;
+/**
+ * A Mapper represents an object that maps data from one Instance type to another.
+ *
+ * It can be used to add, remove, or modify data coming from a Provisioner before
+ * being sent to the user.
+ *
+ * This may be used for anything that modifies the data such as (possibly
+ * conditional) anonymization, or dictionary translations for data submissions.
+ *
+ * @category   Data
+ * @package    Main
+ * @subpackage Data
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+interface Mapper
+{
+    /**
+     * Map returns a copy of $resource that has been modified in some way
+     * without. It must returns a new Instance without having modified the
```

Comment:
Is there something missing here after the first "without"?

Reply:
No, but there's a typo where "returns" should be singular..